### ISSUE TITLE
[Accessibility] Html5 element <nav> does not have the corresponding WAI-ARIA "navigation" role defined.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/wok#89]
Html5 element <nav> used in the content to identify a page Landmark/Section does not have the corresponding WAI-ARIA "navigation" role defined.
